Source/EasyNetQ/Consumer/ConsumerDispatcher.cs

```diff
@@ -11,13 +11,18 @@ public class ConsumerDispatcher : IConsumerDispatcher
         private readonly CancellationTokenSource cancellation = new CancellationTokenSource();
         private readonly BlockingCollection<Action> durableActions = new BlockingCollection<Action>();
         private readonly BlockingCollection<Action> transientActions = new BlockingCollection<Action>();
+        private readonly Thread dispatcherThread;
 
         public ConsumerDispatcher(ConnectionConfiguration configuration)
         {
             Preconditions.CheckNotNull(configuration, "configuration");
 
+            using (ExecutionContext.SuppressFlow())
```

Comment:
Could you do the same for ClientCommandDispatcherSingleton?

Reply:
Should I add logger also? A bit of breaking changes to align internals with `ConsumerDispatcher`.

Reply:
https://github.com/EasyNetQ/EasyNetQ/pull/1046#issuecomment-622237418 (:

Reply:
@sungam3r Yep, please add the same debug logging there. Thanks.

Reply:
OK, no breaking changes at all. For some reason I thought that the logger is being injected in the constructor.

Reply:
Done.